src/emcpy/utils/utils.py

```diff
@@ -172,3 +173,31 @@ def printcolour(text, colour='red'):
 
 
 printcolor = printcolour
+
+
+def tic():
```

Comment:
these function names are not exactly useful... is there a more formal timer/timing module that would be a better thing to include here? like how `logger` works for logging?

Reply:
You're right about the names. These were some functions that I cooked up for myself and thought they might be useful here too. Would you rather that I remove them? I'm not sure what the "formal" method for using a timer is; I would have to do some digging. This isn't an integral part to what I need added. Let me know if I should remove or just update the naming.

Reply:
I suggest removing and we figure out a more robust/comprehensive timer solution later. Does that work for you?